server/memphis_handlers.go

```diff
@@ -200,19 +200,20 @@ func CreateDefaultStation(s *Server, sn StationName, username string) (models.St
 	var newStation models.Station
 	stationName := sn.Ext()
 	newStation = models.Station{
-		ID:                primitive.NewObjectID(),
-		Name:              stationName,
-		RetentionType:     "message_age_sec",
-		RetentionValue:    604800,
-		StorageType:       "file",
-		Replicas:          1,
-		DedupEnabled:      false, // TODO deprecated
-		DedupWindowInMs:   0,     // TODO deprecated
-		CreatedByUser:     username,
-		CreationDate:      time.Now(),
-		LastUpdate:        time.Now(),
-		Functions:         []models.Function{},
-		IdempotencyWindow: 120000,
+		ID:                 primitive.NewObjectID(),
+		Name:               stationName,
+		RetentionType:      "message_age_sec",
+		RetentionValue:     604800,
+		StorageType:        "file",
+		Replicas:           1,
+		DedupEnabled:       false, // TODO deprecated
+		DedupWindowInMs:    0,     // TODO deprecated
+		CreatedByUser:      username,
+		CreationDate:       time.Now(),
+		LastUpdate:         time.Now(),
+		Functions:          []models.Function{},
+		IdempotencyWindow:  120000,
+		DedupConfiguration: true,
```

Comment:
Let's change the name of it to DedupEnabled and make sure you are switching across the entire application

Reply:
This is done.